src/scss/layout/_modal-wrap.scss

```diff
@@ -0,0 +1,83 @@
+.modal-wrap {
+  display: flex;
+  justify-content: center;
+}
+
+.modal-content {
+  color: #f7f7f7;
+  font-size: 1.3rem;
+  min-width: 17rem;
+  max-width: 32rem;
+  background-color: #222222;
+  padding: 2rem 10%;
+  padding-top: 4rem;
+  border-radius: 0.5rem;
+  position: relative;
+  max-height: 70vh;
+  overflow-y: auto;
+
+  h2,
+  h3 {
+    margin: 0;
+    margin-bottom: 1.6rem;
+    text-transform: uppercase;
+    text-align: center;
+  }
+
+  table {
```

Comment:
please do not rely on different css selectors, it will cause unpredictable selector weight conflicts. using classes in combination with BEM / SEM/BIO https://css-tricks.com/combining-the-powers-of-sem-and-bio-for-improving-css/